### ISSUE TITLE
Arkansas Center for Biodiversity Collections at Arkansas State University

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/new-collections#16]
Initial Questionnaire received 9-13-17
[Arkansas State University Initial Questionnaire.pdf](https://github.com/ArctosDB/new-collections/files/1300452/Arkansas.State.University.Initial.Questionnaire.pdf)

**Comment by campmlc** [comment]
emailed to request status 3-6-18

**Comment by Jegelewicz** [comment]
MOU first draft complete. Needs review. https://drive.google.com/file/d/1nRdmz3muoUy2yLlBnIuJDkUSLK3nF_IB/view?usp=sharing
@campmlc @jrdemboski @ccicero

**Comment by Jegelewicz** [comment]
March 27, 2018
MOU draft sent to Travis Marisco <tmarsico@astate.edu> and Kari Harris <kharris@astate.edu>

**Comment by Jegelewicz** [comment]
Email sent today:

Travis and Kari,

I am following up on the Arctos MOU. Please let me know if you need anything from us to get the MOU completed.

Thank you!

**Comment by Jegelewicz** [comment]
|  
-- | --

**Comment by Jegelewicz** [comment]
Sent email to Travis to check progress.

**Comment by Jegelewicz** [comment]
> Hi Teresa,
> 
> Are the changes you mention only to parts 3.1, 3.2, and 3.3 of the document?  I don't think these would cause legal trouble, and we can likely move forward.  We are right at the 100,000 break point...can you please let me know the A-State cost associated with 95,000 specimens, 99,000 specimens, and 105,000 specimens, respectively for year 1?  I want to make sure I am calculating correctly based upon the new fee structure.
> 
> Best,
> 
> Travis
> 
> 
> Travis,
> 

Yes, changes are to the Financial Contributions Section (3), I also added contact info for the person who will sign the document in the Key Personnel section. Financial contributions based on the numbers you gave would be as follows:

  95,000 - $2,782.50
  99,000 - $2,908.50
105,000 - $3,071.25

You can run numbers through the calculator here if you wan to test other specimen record quantities.

Let me know if you need anything else.

**Comment by Jegelewicz** [comment]
Travis,

Just checking on the progress of the Arctos MOU. Do you need anything from me?

Let me know when you have a version that meets all of your requirements so that I can run it by the Arctos Steering Committee before we make a final for signatures.

Thank you!

Teresa J. Mayfield-Meyer
